src/app/beer_garden/db/schemas/garden_schema.py

```diff
@@ -0,0 +1,133 @@
+        _ = self.validate(data, many=many, partial=partial)
+        return super().load(data, many=many, partial=partial)
+
+    def dump(self, obj, many=None, update_fields=True, **kwargs):
+        if isinstance(obj, (list, QuerySet)):
+            many = True
+
+        _ = self.validate(obj, many=many)
+        dumped = super().dump(obj, many=many, update_fields=update_fields, **kwargs)
+
+        return dumped
+
+
+def _port_validator(value):
+    return 1024 <= value < 65535
+
+
+class HttpConnectionParamsSchema(GardenBaseSchema):
+    host = fields.String(required=True)
+    port = fields.Integer(
+        required=True,
+        validate=_port_validator,
+        error_messages={
+            **fields.Field.default_error_messages,
+            **{"validator_failed": "Value out of range for ports"},
+        },
+    )
+    url_prefix = fields.String(required=True, dump_default="/", load_default="/")
+    ca_cert = fields.String(required=False, allow_none=True)
+    ca_verify = fields.Boolean(required=True)
+    client_cert = fields.String(required=False, allow_none=True)
+    client_key = fields.String(required=False, allow_none=True)
+    ssl = fields.Boolean(required=True)
+
+
+class StompSSLParamsSchema(GardenBaseSchema):
+    use_ssl = fields.Boolean(required=True)
+
+
+class StompHeaderSchema(GardenBaseSchema):
+    key = fields.String(required=True)
+    value = fields.String(required=True)
+
+
+class StompConnectionParamsSchema(GardenBaseSchema):
+    ssl = fields.Nested("StompSSLParamsSchema", required=True)
+    headers = fields.List(fields.Nested("StompHeaderSchema"), required=False)
+    host = fields.String(required=True)
+    port = fields.Integer(
+        required=True,
+        validate=_port_validator,
+        error_messages={
+            **fields.Field.default_error_messages,
+            **{"validator_failed": "Value out of range for ports"},
+        },
+    )
+    send_destination = fields.String(required=False, allow_none=True)
+    subscribe_destination = fields.String(required=False, allow_none=True)
+    username = fields.String(required=False, allow_none=True)
+    password = fields.String(required=False, allow_none=True)
+
+
+class GardenConnectionsParamsSchema(GardenBaseSchema):
+    http = fields.Nested("HttpConnectionParamsSchema", allow_none=True)
+    stomp = fields.Nested("StompConnectionParamsSchema", allow_none=True)
+
+
+class GardenSchema(GardenBaseSchema):
+    id = fields.Str(allow_none=True)
+    name = fields.Str(allow_none=False)
+    status = fields.Str(allow_none=True)
+    status_info = fields.Nested(StatusInfoSchema, allow_none=True)
+    connection_type = fields.Str(allow_none=False)
+    connection_params = fields.Nested(GardenConnectionsParamsSchema, allow_none=True)
+    namespaces = fields.List(fields.Str(), allow_none=True)
+    systems = fields.Nested(SystemSchema, many=True, allow_none=True)
+
+    @post_load
+    def make_object(self, data):
+        return BrewtilsGarden(**data)
```

Comment:
This worries me a bit, for reasons that I elaborated on in the comment on the `from_brewtils` changes.